tests/specs/lemmas.k

```diff
@@ -14,6 +14,7 @@ module LEMMAS
     rule #sizeByteArray(#buf(N, _)  )  => N                                               [simplification]
     rule #sizeByteArray(BUF1 ++ BUF2)  => #sizeByteArray(BUF1) +Int #sizeByteArray(BUF2)  [simplification]
     rule #sizeByteArray(#drop(N, BUF)) => maxInt(#sizeByteArray(BUF) -Int N, 0)           [simplification]
+    rule #sizeByteArray(M [ N := BUF ]) => maxInt(#sizeByteArray(M), #sizeByteArray(BUF) +Int N)           [simplification]
```

Comment:
Formatting. Line up the `=>` with other rules, and line up the `[simplification]`.